From 627aba7c2f7e92f759b8f013522ecb27f79ffb4f Mon Sep 17 00:00:00 2001
From: Andrew Lunn <andrew@lunn.ch>
Date: Mon, 5 May 2025 17:53:56 -0500
Subject: [PATCH 04/29] net: phy: maxio: Add missing static

Some of these functions will soon disappear, but it at last now compiles
without warnings.

Signed-off-by: Andrew Lunn <andrew@lunn.ch>
---
 drivers/net/phy/maxio.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/phy/maxio.c b/drivers/net/phy/maxio.c
index 5d3d2f029..e465f08af 100755
--- a/drivers/net/phy/maxio.c
+++ b/drivers/net/phy/maxio.c
@@ -26,7 +26,7 @@
 #define MAXIO_MAE0621A_WORK_STATUS_REG   0x1d
 
 
-int maxio_read_paged(struct phy_device *phydev, int page, u32 regnum)
+static int maxio_read_paged(struct phy_device *phydev, int page, u32 regnum)
 {
 	int ret = 0, oldpage;
 
@@ -40,7 +40,7 @@ int maxio_read_paged(struct phy_device *phydev, int page, u32 regnum)
 	return ret;
 }
 
-int maxio_write_paged(struct phy_device *phydev, int page, u32 regnum, u16 val)
+static int maxio_write_paged(struct phy_device *phydev, int page, u32 regnum, u16 val)
 {
 	int ret = 0, oldpage;
 
@@ -214,7 +214,7 @@ static int maxio_mae0621a_resume(struct phy_device *phydev)
 	return ret;
 }
 
-int maxio_mae0621a_suspend(struct phy_device *phydev)
+static int maxio_mae0621a_suspend(struct phy_device *phydev)
 {
 	genphy_suspend(phydev);
 	phy_write(phydev, MAXIO_PAGE_SELECT ,0);
-- 
2.39.5

